autopilot-daemon/pkg/handlers/healthchecks.go

```diff
@@ -22,11 +22,35 @@ func PeriodicCheckTimer() {
 }
 
 func InvasiveCheckTimer() {
-	klog.Info("Trying to run an intrusive check")
+	klog.Info("Trying to run an invasive check")
 	utils.HealthcheckLock.Lock()
 	defer utils.HealthcheckLock.Unlock()
 	if utils.GPUsAvailability() {
-		utils.CreateJob("dcgm")
+		klog.Info("Begining invasive health checks, updating node label =TESTING for node ", os.Getenv("NODE_NAME"))
+		label := `
+		{
+			"metadata": {
+				"labels": {
+					"autopilot.ibm.com/gpuhealth": "TESTING"
```

Comment:
I didn't mention in the description of the issue, so totally my bad!! But we also need to take the testing label out of the way after the test is completed.

Reply:
I thought of this. Will the resulting label (e.g., ERR or PASS) overwrite the TESTING label?

Reply:
Should `"autopilot.ibm.com/gpuhealth"` be left alone? The PR, as it stands, makes sense to me. That is, `gpuhealth` has three states: `ERR`, `PASS` and `TESTING`. When `TESTING` it is neither `ERR` or `PASS`.

I do like the idea of the timestamp. But where would that go? `ERR_30-08-2024_09:37:51` ?

Reply:
hmmm ok, I don't have strong arguments against using `TESTING` on `autopilot.ibm.com/gpuhealth`. But I will oppose to the timestamp because it might complicate things with label selector for affinity/anti-affinity, or not making it work at all

Reply:
Does the label itself have a creation time?

Reply:
No, it doesn't. But for this particular one we really don't care